pageserver/src/layered_repository.rs

```diff
@@ -465,6 +472,20 @@ impl From<LayeredTimelineEntry> for RepositoryTimeline<LayeredTimeline> {
 
 /// Private functions
 impl LayeredRepository {
+    pub fn get_tenant_conf(&self) -> TenantConf {
```

Comment:
IMO using getters instead of merging configs every time we need them is a bit cleaner. E g in the compaction thread we're using only `compaction_period`

Reply:
Agree. I'll implement getters.
Though, we still need get_tenant_conf() to implement show.